optaplanner-operator/school-timetabling-parent/pom.xml

```diff
@@ -0,0 +1,161 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
This school-timetabling project is an implementation detail of the `optaplanner-operator-integration-tests` module, thus it should be located there and not in the same directory as the operator itself.